packages/IMAPClient-UI.package/ICAddressBookDialog.class/instance/sortContactsBy..st

```diff
@@ -0,0 +1,8 @@
+action
+sortContactsBy: aMethod
+	
+	(self sortMethod = aMethod) ifTrue: [self reverseContacts] ifFalse: [
```

Comment:
No brackets around `self sortedMethod = aMethod` needed. Also, maybe put the `ifFalse` statement in a new line